src/main/java/InfrastructureManager/EdgeNode.java

```diff
@@ -4,17 +4,20 @@
     private String id;
     private String ipAddress;
     private boolean connected;
+    private int port;
```

Comment:
yeah, I agree if it's used as ip:port in all other places then it might be more sensible to let them be and separate them in the application as it is a specific requirement within the application